css/styles.css

```diff
@@ -0,0 +1,602 @@
+  height: 100vh;
+  text-align: center;
+  transition: 0.3s;
+  padding-top: 100px;
+  z-index: 1;
+}
+
+.contact {
+  text-align: center;
+  margin-top: 30px;
+}
+
+.contact a {
+  color: #272a31;
+  cursor: pointer;
+  font-weight: bold;
+  text-decoration: none;
+}
+
+.nav-link a {
+  text-decoration: none;
+  color: #272a31;
+  font-weight: 700;
+  transition: 0.3s;
+  font-size: 1.1rem;
+}
+
+.nav-link a:hover {
+  color: #ed5342;
+}
+
+li:nth-child(6) {
+  border-width: 3px;
+  border-style: solid;
+  border-image: linear-gradient(to right, #ec5242, #272a31) 1;
+  padding: 7px;
+}
+
+section.hero {
+  display: flex;
+  flex-direction: column;
+  padding: 40px 30px;
+  height: 100vh;
+  margin-top: 60px;
+}
+
+.hero-text {
+  font-family: "Lato", sans-serif;
+  font-size: 1.3rem;
+  font-weight: 700;
+  color: #ed5342;
+}
+
+.hero-header {
+  font-family: inherit;
+  font-size: 2.5rem;
+  font-weight: bolder;
+  color: #ed5342;
+  text-transform: uppercase;
+  margin-bottom: 30px;
+}
+
+.text {
+  background-image: url(../images/text-bg-1.jpg);
+  background-repeat: repeat;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+}
+
+.description {
+  font-size: 1rem;
+  font-family: inherit;
+  padding: 15px;
+  margin-top: 10px;
+  line-height: 150%;
+  opacity: 0.7;
+}
+
+.header-h2 {
+  font-family: inherit;
+  text-transform: uppercase;
+  font-weight: bolder;
+}
+
+.main-program {
+  background-image: url(../images/program-bg.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  padding: 2rem;
+}
+
+.program-heading {
+  color: #fff;
+  font-size: 1.8rem;
+  background-color: #272932;
+  text-align: center;
+  padding-top: 30px;
+  position: relative;
+  margin-bottom: 3rem;
+}
+
+.program-heading::after {
+  content: "";
+  height: 3px;
+  width: 20%;
+  background: linear-gradient(to right, #d3d3d3, #f24432);
+  position: absolute;
+  margin: auto;
+  bottom: -5px;
+  left: 0;
+  right: 0;
+}
+
+.card-container {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  gap: 1rem;
+  justify-content: center;
+  align-items: center;
+}
+
+.card {
+  background-color: #b3b3b351;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 0.5rem;
+  width: 100%;
+  padding: 0.5rem;
+}
+
+.card:hover {
+  border: 1px solid #ed5342;
+}
+
+.card-header {
+  color: #ed5342;
+  font-weight: 700;
+  font-family: inherit;
+  font-size: 1.1rem;
+}
+
+.card-text {
+  color: #fff;
+  font-weight: 700;
+  font-family: inherit;
+}
+
+.team {
+  padding: 3rem 4rem;
+}
+
+.program-heading.group {
+  color: #272a31;
+  background-color: transparent;
+}
+
+.team-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 15px;
+}
+
+.frame img {
+  width: 10rem;
+  height: 11rem;
+  object-fit: cover;
+}
+
+.individual {
+  display: flex;
+  gap: 15px;
+}
+
+.team-details {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+}
```

Comment:
- [ ] Hey, please let's improve our design in the `mobile version` a little to be close to the design template as the aim of this project is to help make a total clone of an existing design, thereby paying attention to details is a skill that will make you stand out in the job market. see the suggestion below:

*original design*
![Screenshot from 2022-10-19 08-43-13](https://user-images.githubusercontent.com/59397159/196631660-d3e4e2a8-ea62-48fb-ae72-2c425966cee2.png)

*your design*
![Screenshot from 2022-10-19 08-45-03](https://user-images.githubusercontent.com/59397159/196631649-5261937a-162d-48fc-83a3-3d85823dc183.png)

*your design after I edited*
![Screenshot from 2022-10-19 08-52-53](https://user-images.githubusercontent.com/59397159/196631645-ad0b13f7-a48f-47d0-8a33-a83419b994b8.png)

- As you can from those designs, the content on your design is not that close compared to the original design
   - You can reduce the font size,
   - Remove the gap in `.team-details`,
   - You can make the height/width of the image `8rem`,
   - You can also reduce the description of the team members,*[If you do this, your design will be very close :100:`]*
   - All these are just little tweaks around to improve the design.
- [ ] Please, make a few adjustments to improve your design and also, to improve the user experience of your website.